packages/contracts/deployments/engine/V3/partners/jeff-davis-flex/DEPLOYMENT_LOGS.log

```diff
@@ -45,3 +45,99 @@ Contracts metadata upsert input:
+[INFO] Deploying to network: mainnet
+[INFO] Deploying to environment: mainnet
+[INFO] New Admin ACL AdminACLV1 deployed at address: 0x732C060EbF93cfF0314CABCCc1BF04904B53E9fB
+[INFO] Using shared Randomizer at 0x13178A7a8A1A9460dBE39f7eCcEbD91B31752b91
+[INFO] Core GenArt721CoreV3_Engine_Flex deployed at 0x5306e34B7437200E0189CbC5F80B0990E49DCBE7
+[INFO] Registered core contract 0x5306e34B7437200E0189CbC5F80B0990E49DCBE7 on core registry 0x2eE7B9bB2E038bE7323A119701A191c030A61ec6
+[INFO] Updated the Minter Filter on the Core contract to 0xa2ccfE293bc2CDD78D8166a82D1e18cD2148122b.
+[INFO] Skipping adding placeholder initial project.
+[INFO] Skipping update of render provider split percentage primary, since it is already equal to the value of 10.
+[INFO] Skipping update of render provider split percentage secondary, since it is already equal to the value of 250.
+[INFO] Transferred superAdmin role on adminACL to 0x1D627Fe53e605f37ec5F6a58EBcE1f3A39882D68.
+[INFO] Verifying core contract contract deployment...
+Successfully submitted source code for contract
+contracts/engine/V3/GenArt721CoreV3_Engine_Flex.sol:GenArt721CoreV3_Engine_Flex at 0x5306e34B7437200E0189CbC5F80B0990E49DCBE7
+for verification on the block explorer. Waiting for verification result...
+
+[ACTION] Save the following constructor args config file to a constructor-args.js file, then verify core contract deployment with:
+module.exports = [
+        "Davis Editions", // name
+        "DE", // ticker
+        "0x036F3D03C1ccdde1878F01607922EA12110Ee9Bd", // render provider
+        "0x1D627Fe53e605f37ec5F6a58EBcE1f3A39882D68", // platform provider
+        "0x13178A7a8A1A9460dBE39f7eCcEbD91B31752b91", // randomizer
+        "0x732C060EbF93cfF0314CABCCc1BF04904B53E9fB", // admin acl
+        0, // starting project id
+        false, // auto approve artist split proposals
+      ];
+yarn hardhat verify --network mainnet --constructor-args constructor-args.js 0x5306e34B7437200E0189CbC5F80B0990E49DCBE7
+[INFO] Verifying AdminACL contract deployment...
+[WARN] Etherscan verification of AdminACL failed: UnexpectedError: An unexpected error occurred during the verification process.
+Please report this issue to the Hardhat team.
+Error Details: Failed to send contract verification request.
+Endpoint URL: https://api.etherscan.io/api
+Reason: The Etherscan API responded that the address 0x732C060EbF93cfF0314CABCCc1BF04904B53E9fB does not have bytecode.
+This can happen if the contract was recently deployed and this fact hasn't propagated to the backend yet.
+Try waiting for a minute before verifying your contract. If you are invoking this from a script,
+try to wait for five confirmations of your contract deployment transaction before running the verification subtask.
+[ACTION] Verify AdminACL contract deployment with:
+yarn hardhat verify --network mainnet 0x732C060EbF93cfF0314CABCCc1BF04904B53E9fB 
```

Comment:
@lindsgil it looks like this still needs verification. maybe an etherscan api key issue?

Reply:
good catch! verified ✅  https://etherscan.io/address/0x732C060EbF93cfF0314CABCCc1BF04904B53E9fB#code